gnovm/tests/files/zrealm_crossrealm20.gno

```diff
@@ -0,0 +1,15 @@
+// PKGPATH: gno.land/r/crossrealm_test
+package crossrealm_test
+
+import (
+	crossrealm "gno.land/r/demo/tests/crossrealm"
+)
+
+var b0 crossrealm.Bar // run declarations
+
+func main() {
+	print(".")
+}
+
+// Error:
+// should not happen while attempting to attach new real object from external realm
```

Comment:
forbid attach object by value with type defined in external-realm.

Reply:
This panic looks correct to me,  because `Bar` is a struct so initialization would create a new value, which would be attached to `crossrealm_test` but it cannot.